dedoc/api/static/html_rus/format_description.html

```diff
@@ -91,11 +91,40 @@ <h3 id="TreeNode"> TreeNode. Древовидная структура доку
 <h4 id="Annotation"> Annotation. Аннотации текста.</h4>
 <ol>
     <li><strong> start </strong> : int (обязательное поле) - индекс начала аннотации.</li>
-    <li><strong> end </strong> : int (обязательное поле) - индекс конца аннотации.</li>
+    <li><strong> end </strong> : int (обязательное поле) - индекс конца аннотации.
+        Равен индексу последнего символа (относящегося к данной аннотации) + 1.
+        Например, если в строке проаннотирован только первый символ, то start = 0, end = 1;
+        если проаннотирована вся строка s, то start = 0, end = длина s. </li>
     <li><strong> name </strong> : string (обязательное поле) - тип аннотации (размер шрифта, курсив и т.д.).</li>
-    <li><strong> value </strong> : str (обязательное поле) - значение аннотации.</li>
+    <li><strong> value </strong> : str (обязательное поле) - значение аннотации
+        (подробнее в <a href="#ConcreteAnnotations">ConcreteAnnotations</a>).</li>
 </ol>
 
+<h4 id="ConcreteAnnotations"> Concrete annotations. Конкретные виды аннотаций.</h4>
+<ul>
+    <li><strong> AlignmentAnnotation </strong> : выравнивание текста. Значение поля name = "alignment".
+        Значением поля value может быть "left" (выравнивание по левому краю), "right" (выравнивание по правому краю),
+        "both" (выравнивание по обоим краям), "center" (выравнивание по центру). </li>
+    <li><strong> BoldAnnotation </strong> : жирность текста. Значение поля name = "bold".
+        Значением поля value может быть "True", если шрифт жирный и "False" иначе. </li>
+    <li><strong> IndentationAnnotation </strong> : отступ от левого края страницы.
+        Для документов в формате docx хранится отступ от внутренних границ документа
+        (не учитывается отступ от края страницы). Значение поля name = "indentation".
+        Значением поля value может быть любое действительное число, сконвертированное в строку.
+        Для документов в формате docx отступ измеряется в двадцатых пункта (1/1440 дюйма). </li>
+    <li><strong> ItalicAnnotation </strong> : выделен ли текст курсивом. Значение поля name = "italic".
+        Значением поля value может быть "True", если текст выделен курсивом и "False" иначе. </li>
+    <li><strong> SizeAnnotation </strong> : размер шрифта в пунктах (1/72 дюйма). Значение поля name = "size".
+        Значением поля value может быть любое действительное число, сконвертированное в строку.</li>
+    <li><strong> StyleAnnotation </strong> : название стиля, примененного к тексту, например, "heading 1".
+        Значение поля name = "style".
+        Значением поля value может быть любая строка.</li>
+    <li><strong> Table Annotation </strong> : является ли строка ссылкой на таблицу. Значение поля name = "table".
+        Значение поля value = "table " + уникальный идентификатор таблицы. </li>
```

Comment:
name it is not "table" + table_name but only table name.

Reply:
done